serverless/README.md

```diff
@@ -0,0 +1,186 @@
+---
+layout: pattern
+title: serverless
+folder: serverless
+permalink: /patterns/serverless/
+categories: Architectural
+tags:
+ - Java
+ - Difficulty-Intermittent
```

Comment:
Difficulty-Intermediate?

Reply:
@iluwatar I feel like the difficulty level is Intermediate based on the complexity to adopt, but let me know if your thoughts?

Reply:
I mean literally, Difficulty-Intermittent is wrong...

Reply:
@iluwatar good catch, I think I copied the initial template from EIP patter, yup...will fix that with an other PR.